src/scenes/home/team/executiveStaff.js

```diff
@@ -20,7 +20,7 @@ const COO = {
 };
 
 const CCMO = {
-  name: 'Jennifer Weiderman',
+  name: 'Jennifer Weideman',
   role: 'CCMO',
   slackUsername: '@jenn',
```

Comment:
slack now allows both `jenn` and `Jenn` to be valid. But if there's not a user for each it'll ping the right one if you use the other. please change this to `Jenn`

Reply:
Also if possible would you update these:
David Reis: "DavidR"
Amy Tan:  "Amy Tan"
Kelly MacLeod: "Kelly"  "kelly" is another person. 
Jameel Martin: Name is actually "Jameel Matin"

Reply:
Sure, will do!